autoload/battery/backend/linux.vim

```diff
@@ -1,15 +1,15 @@
 " Ref: https://github.com/lambdalisue/battery.vim/issues/7
-let s:Job = vital#battery#import('System.Job')
-let s:ac_online = get(glob('/sys/class/power_supply/AC*/online', 0, 1), 0, '')
-let s:bat_capacity = get(glob('/sys/class/power_supply/BAT*/capacity', 0, 1), 0, '')
+let s:bat_files = glob('/sys/class/power_supply/{CMD*,BAT*,battery}/{status,capacity}', 0, 1)
+let s:bat_status = get(s:bat_files, 0, '')
+let s:bat_capacity = get(s:bat_files, 1, '')
```

Comment:
`s:bat_capacity` might get wrong path when there are several match on the first pattern.

Assume that you have

```
├── a
│   ├── a
│   ├── b
│   └── c
└── b
    ├── a
    ├── b
    └── c
```

And then

```
:echo glob('./{a,b}/{a,b}', 0, 1)
['./a/a/', './a/b/', './b/a/', './b/b/']
```

Probably, it's better to use glob in each variables like

```suggestion
let s:bat_status = get(glob('/sys/class/power_supply/{CMD*,BAT*,battery}/status', 0, 1), 0, '')
let s:bat_capacity = get(glob('/sys/class/power_supply/{CMD*,BAT*,battery}/capacity', 0, 1), 0, '')
```